docs/.pages

```diff
@@ -7,3 +7,4 @@ nav:
   - contributing
   - datasets
   - support
+  - Competitions: autoware-competitions/index.md
```

Comment:
It looked like when the `index.md` file is specified, the link in the nav bar is not working: https://autowarefoundation.github.io/autoware-documentation/pr-382/autoware-competitions/index.md

Reply:
resolved in https://github.com/autowarefoundation/autoware-documentation/pull/382/commits/cef9e8f26303bbde4d0c84127d8a3460f59de90a !